style.css

```diff
@@ -64,6 +64,10 @@ h3 {
     color: #6d7884;
 }
 
+h3 span {
```

Comment:
Do you want to use a child selector `>` so that you aren't styling any `span` inside an `h3`?

Reply:
Good idea!